common/src/IO/TextureLoader.cpp

```diff
@@ -58,6 +59,9 @@ namespace TrenchBroom {
             } else if (textureConfig.format.format == "idwal") {
                 TextureReader::PathSuffixNameStrategy nameStrategy(2, true);
                 return new IdWalTextureReader(nameStrategy, loadPalette(textureConfig));
+            } else if (textureConfig.format.format == "image") {
+                TextureReader::PathSuffixNameStrategy nameStrategy(3, true);
```

Comment:
So far classic games will not use this feature anyway, while modern games(maybe DarkPlaces-based) often have similar texture names in different folders, so I thought that it would be safer to store "more unique" texture name. Not sure it is that necessary though.